backend/app/infrastructure/routes/user.py

```diff
@@ -316,3 +331,26 @@ def get(self, epoch: int):
                 for user_address, uq_score in uq_scores
             ]
         }
+
+
+@ns.route("/<string:user_address>/winnings/<int:epoch>")
+@ns.doc(
+    params={
+        "epoch": "Epoch number",
```

Comment:
Why epoch number? Raffles are not bound in any way to epochs and time winnings are locked for can be of any length.
Adding this `epoch` parameter here would require FE to iterate over all epochs to get all winnings.

Maybe there is something I don't see here.

Reply:
Sure, changed.